Podcast/app/src/main/java/br/ufpe/cin/if710/podcast/domain/XmlFeedParser.java

```diff
@@ -72,8 +72,8 @@ public static ItemFeed readItem(XmlPullParser parser) throws XmlPullParserExcept
             if (name.equals("title")) {
                 title = readData(parser, "title");
             }
-            else if (name.equals("link")) {
-                link = readData(parser, "link");
+            else if (name.equals("guid")) {
+                link = readData(parser, "guid");
```

Comment:
Fixing tag name, according to the XML.

<img width="518" alt="screen shot 2017-10-15 at 2 20 16 pm" src="https://user-images.githubusercontent.com/1585655/31587208-005ade36-b1b4-11e7-9ea7-ecea58a5320d.png">